Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "ObjectFile"
 uuid = "d8793406-e978-5875-9003-1fc021f44a92"
 authors = ["Elliot Saba <staticfloat@gmail.com>"]
-version = "0.3.7"
+version = "0.4.0"
 
 [deps]
```

Comment:
Can you rebase and bump the minimum version of Julia to v1.6?  That should fix the tests.